clang/lib/Index/IndexUnitReader.cpp

```diff
@@ -433,7 +445,7 @@ IndexUnitReader::createWithFilePath(StringRef FilePath, std::string &Error) {
     return nullptr;
   }
 
-  std::unique_ptr<IndexUnitReaderImpl> Impl(new IndexUnitReaderImpl());
+  std::unique_ptr<IndexUnitReaderImpl> Impl(new IndexUnitReaderImpl(Remapper));
```

Comment:
```suggestion
  auto Impl = std::make_unique<IndexUnitReaderImpl>(Remapped);
```

Sorry I missed this on my first run through